doc/index.rst

```diff
@@ -10,7 +10,7 @@
         </h2>
     </div>
 
-.. include:: ../README.rst
+.. include:: ../README.md
     :start-after: doc-index-start-after
     :end-before: doc-index-end-before
```

Comment:
~~~suggestion
```{include} ../README.md
:start-after: doc-index-start-after
:end-before: doc-index-end-before
```
~~~

Reply:
I tried this idea, but unfortunately, this also does not work:

| GitHub | Website |
| --- | --- |
| ![include_md_in_rst_GitHub](https://github.com/GenericMappingTools/pygmt/assets/94163266/f9136de4-4c9c-424b-9752-22a7c3907187) | ![include_md_in_rst_Webside](https://github.com/GenericMappingTools/pygmt/assets/94163266/93670963-9074-418c-9153-1254e9271f59) |

Reply:
perhaps you have to convert index.rst to index.md

Reply:
I will try this, but I will be quite busy in the next three weeks. Thus, I am not sure if I will have the time to do this in the next days 🙁.

Reply:
Thanks @seisman for doing this and completing this PR.